tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -398,16 +398,91 @@ internal TracerSettings(IConfigurationSource? source, IConfigurationTelemetry te
 
             StatsComputationInterval = config.WithKeys(ConfigurationKeys.StatsComputationInterval).AsInt32(defaultValue: 10);
 
+            OtelMetricsExporter = config
+                        .WithKeys(ConfigurationKeys.OpenTelemetry.MetricsExporter)
+                        .AsString(defaultValue: "otlp");
+
+            OtelMetricsExporterEnabled = string.Equals(OtelMetricsExporter, "otlp", StringComparison.OrdinalIgnoreCase);
+
             var otelRuntimeMetricsEnabled = config
-                                          .WithKeys(ConfigurationKeys.OpenTelemetry.MetricsExporter)
-                                          .AsBoolResult(
-                                               value => string.Equals(value, "none", StringComparison.OrdinalIgnoreCase)
-                                                            ? ParsingResult<bool>.Success(result: false)
-                                                            : ParsingResult<bool>.Failure());
+                            .WithKeys(ConfigurationKeys.OpenTelemetry.MetricsExporter)
```

Comment:
@andrewlock is this going to cause multiple telemetry calls? If we need to do only one call, I think we may be able to add this as a validator on the first call to only accept `none` and `otlp` as valid

Reply:
Also discussed [here](https://github.com/DataDog/dd-trace-dotnet/pull/7420#discussion_r2325553989) and TLDR I am not validating `otlp` as a value when trying to check it for Runtime Metrics, Andrew and I also discussed this `otlp` value not doing anything for Runtime Metrics before so let me know what you guys think, or if we should revisit this override scenario...

Reply:
I've added some updates based on this convo and some other offline ones were some tests broke and we discussed  how to get default telemetry and proper logging:

https://github.com/DataDog/dd-trace-dotnet/pull/7420/files#diff-8afc30ae77b35082c438827c3ab04d7dce486226317aeef23e40e7bd62ba2b75R401